src/table/TablePaginationActions.jsx

```diff
@@ -43,38 +39,38 @@ export default function TablePaginationActions(props) {
   return (
     <div className={classes.root}>
       <IconButton
-        onClick={!onFirstPage && handleFirstPageButtonClick}
-        aria-disabled={onFirstPage}
+        onClick={handleFirstPageButtonClick}
```

Comment:
So nothing happens if you are on first page and click now?

Reply:
yes, nothing happens